packages/salesforcedx-vscode-aura/test/extension.test.ts

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { expect } from 'chai';
+import * as vscode from 'vscode';
+
+const PERFECT_MATCH = 10;
+
+async function matchExtensionAsHtml(extension: string) {
+    const doc = await vscode.workspace.openTextDocument(vscode.Uri.parse(`untitled:fake/path/doc.${extension}`));
+    expect(vscode.languages.match({ language: 'html' }, doc)).to.equal(PERFECT_MATCH);
+}
+
+describe('Lightning file association', () => {
+    it('Should support .app association', async () => {
+        await matchExtensionAsHtml('.app');
+    });
+
+    it('Should support .cmp association', async () => {
+        await matchExtensionAsHtml('.cmp');
+    });
+
+    it('Should support .design association', async () => {
+        await matchExtensionAsHtml('.design');
+    });
+
+    it('Should support .evt association', async () => {
+        await matchExtensionAsHtml('.evt');
+    });
+
+    it('Should support.intf association', async () => {
+        await matchExtensionAsHtml('.intf');
+    });
+
+    it('Should support .auradoc association', async () => {
+        await matchExtensionAsHtml('.auradoc');
+    });
+
+    it('Should support .tokens association', async () => {
+        await matchExtensionAsHtml('.tokens');
+    });
+});
+
+describe('Test commands', () => {
+    let coreExtension: vscode.Extension<any>;
+    let auraExtension: vscode.Extension<any>;
+
+    before(async () => {
+        if (vscode.workspace.rootPath) {
```

Comment:
`rootPath` is deprecated, you can replace it with `vscode.workspace.workspaceFolders![0]`.